casepro/backend/junebug.py

```diff
@@ -115,7 +175,28 @@ def pull_contacts(
             tuple of the number of contacts created, updated, deleted and
             ignored
         """
-        return (0, 0, 0, 0)
+        identity_store = self.identity_store
+
+        # all identities created in the Identity Store in the time window
+        new_identities = identity_store.get_identities(
+            created_at__gte=modified_after)
+
+        # all identities modified in the Identity Store in the time window
+        modified_identities = identity_store.get_identities(
+            updated_at__gte=modified_after,
+            updated_at__lte=modified_before)
+
+        identities_to_update = chain(modified_identities, new_identities)
+
+        # all identities deleted in the Identity Store in the time window
+        deleted_identities = identity_store.get_identities(
+            optout__optout_type='forget', updated_at__gte=modified_after,
+            updated_at__lte=modified_before)
+
+        # the method expects fetches not lists so I faked it
+        return sync_local_to_changes(
+            org, IdentityStoreContactSyncer(), [identities_to_update],
+            [deleted_identities], progress_callback)
```

Comment:
I don't think we want the square brackets here -- or at least it seems odd to have a list of iterators.